src/BloomExe/FontProcessing/FontMetadata.cs

```diff
@@ -0,0 +1,351 @@
+						//{
+						//	try
+						//	{
+						//		var sfntName = face.GetSfntName(i);
+						//		...
+						//	}
+						//	catch (Exception ex)
+						//	{
+						//	}
+						//}
+					}
+				}
+			}
+			catch (Exception ex)
+			{
+				Console.WriteLine("SharpLib EXCEPTION for {0}: {1}", group.Normal, ex);
+			}
+			try
+			{
+				var process = new Process
+				{
+					StartInfo = new ProcessStartInfo()
+					{
+						FileName = "/usr/bin/otfinfo",
+						Arguments = $"-i \"{group.Normal}\"",
+						UseShellExecute = false,
+						CreateNoWindow = true,
+						RedirectStandardOutput = true,
+						RedirectStandardError = true,
+					},
+				};
+				process.Start();
+				process.WaitForExit();
+				var standardOutput = process.StandardOutput.ReadToEnd();
+				var standardError = process.StandardError.ReadToEnd();
+				if (process.ExitCode == 0 && standardError.Length == 0)
+					ParseOtfinfoOutput(standardOutput);
+				else
+					Console.WriteLine("oftinfo -i \"{0}\" returned {1}.  Standard Error =\n{2}", group.Normal, process.ExitCode, standardError);
+			}
+			catch (Exception e)
+			{
+				Console.WriteLine("otfinfo -i \"{0}\" threw an exception: {1}", group.Normal, e);
+			}
+#else
+			GlyphTypeface gtf = null;
+			try
+			{
+				gtf = new GlyphTypeface(new Uri("file:///" + group.Normal));
+				var english = System.Globalization.CultureInfo.GetCultureInfo("en-US");
+				version = gtf.VersionStrings[english];
+				copyright = gtf.Copyrights[english];
+				var embeddingRights = gtf.EmbeddingRights;
+				switch (embeddingRights)
+				{
+					case FontEmbeddingRight.Editable:
+					case FontEmbeddingRight.EditableButNoSubsetting:
+						fsType = "Editable";
+						break;
+					case FontEmbeddingRight.Installable:
+					case FontEmbeddingRight.InstallableButNoSubsetting:
+						fsType = "Installable";
+						break;
+					case FontEmbeddingRight.PreviewAndPrint:
+					case FontEmbeddingRight.PreviewAndPrintButNoSubsetting:
+						fsType = "Print and preview";
+						break;
+					case FontEmbeddingRight.RestrictedLicense:
+						fsType = "Restricted License";
+						break;
+					case FontEmbeddingRight.EditableButNoSubsettingAndWithBitmapsOnly:
+					case FontEmbeddingRight.EditableButWithBitmapsOnly:
+					case FontEmbeddingRight.InstallableButNoSubsettingAndWithBitmapsOnly:
+					case FontEmbeddingRight.InstallableButWithBitmapsOnly:
+					case FontEmbeddingRight.PreviewAndPrintButNoSubsettingAndWithBitmapsOnly:
+					case FontEmbeddingRight.PreviewAndPrintButWithBitmapsOnly:
+						fsType = "Bitmaps Only";
+						break;
+				}
+				designer = gtf.DesignerNames[english];
+				designerURL = gtf.DesignerUrls[english];
+				license = gtf.LicenseDescriptions[english];
+				manufacturer = gtf.ManufacturerNames[english];
+				manufacturerURL = gtf.VendorUrls[english];
+				trademark = gtf.Trademarks[english];
+			}
+			catch (Exception)
+			{
+				// file is somehow corrupt or not really a font file? Just ignore it.
```

Comment:
Seems like at this point, we should be labelling the font in such a way as to indicate that we ran into an Error? E.g. can we set `determinedSuitabilityNotes`?